signatures/php/fruit.db

```diff
@@ -26,6 +26,8 @@ file_get_contents[[:space:]]*\(.*\$(_ENV|_GET|_POST|_COOKIE|_REQUEST|_SERVER|HTT
 (is_dir|file_exists|unlink)[[:space:]]*\(\"?\$(_ENV|_GET|_POST|_COOKE|_REQUEST|_SERVER|HTTP|http).*\)
 show_source[[:space:]]*\(.*\$(_ENV|_GET|_POST|_COOKIE|_REQUEST|_SERVER|HTTP|http).*\)
 preg_replace[[:space:]]*\([\'"](.).*\1[igsu]*e
+preg_match[[:space:]]*\(.*
```

Comment:
Sorry, the fruit rules are for high value (remote code exec type vulnerabilites) with *low* false positive rates, these two rules do not belong here.

Reply:
What about moving ```preg_match``` to ```php/todo.db``` and ```scandir``` to ```php/info.db```

Reply:
Afaik there isn't any significant vulnerabilities with preg_match, could you give an example? scandir should probably be in streams.db

Reply:
The title of the commit for ```preg_match``` is Regex Bypass which indicates Regex Filter/Restrictions Bypass.

Let's think about a file upload scenario.
The following code line limits the uploaded file type to `jpg`. 
```preg_match("/\.jpg/", $file)```

However, it's possible to upload a file named shell.jpg.php which can cause remote code execution.
It is a well-know vulnerability.

Here is a detailed reference: https://github.com/attackercan/regexp-security-cheatsheet

Reply:
You'd need to tick a few more boxes than just the regex and in array is also a common pattern, but fair enough